compositor_integration_tests/src/common.rs

```diff
@@ -38,3 +40,30 @@ pub fn output_dump_from_disk<P: AsRef<Path>>(path: P) -> Result<Bytes> {
     let bytes = fs::read(output_path).context("Failed to read output dump")?;
     Ok(Bytes::from(bytes))
 }
+
+pub fn split_rtp_packet_dump(dump: Bytes, split_at_pts: Duration) -> Result<(Bytes, Bytes)> {
+    let mut read_bytes = 0;
+    let mut start_pts = None;
+
+    while read_bytes < dump.len() {
+        let packet_len = u16::from_be_bytes([dump[read_bytes], dump[read_bytes + 1]]) as usize;
+        read_bytes += 2;
+
+        let packet =
+            rtp::packet::Packet::unmarshal(&mut dump.slice(read_bytes..(read_bytes + packet_len)))?;
+        read_bytes += packet_len;
+
+        let packet_pts = match packet.header.payload_type {
+            96 => Duration::from_secs_f64(packet.header.timestamp as f64 / 90000.0),
+            97 => Duration::from_secs_f64(packet.header.timestamp as f64 / 48000.0),
+            _ => continue,
+        };
```

Comment:
I think we want to return an error when LC sends RTP packet with a payload type different than 96 or 97